lib/v2/arcteryx/radar.js

```diff
@@ -10,4 +10,15 @@ module.exports = {
             },
         ],
     },
+    'regear.arcteryx.com': {
+        _name: 'Arcteryx',
+        '.': [
```

Comment:
The object key can only match the domain name without hostname.
```suggestion
    'arcteryx.com': {
        _name: 'Arcteryx',
        regear: [
```